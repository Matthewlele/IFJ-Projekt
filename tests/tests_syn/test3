# chyba! nelze definovat s velkym pismenem na zacatku
Var = 6
