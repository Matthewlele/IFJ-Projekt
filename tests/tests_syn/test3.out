expected SCANNER ERR: row 2
